packages/dynamite/dynamite/pubspec.yaml

```diff
@@ -5,6 +5,7 @@ environment:
   sdk: '>=3.1.0 <4.0.0'
 
 dependencies:
+  basics: ^0.10.0
```

Comment:
This feels a bit like the Javascript way of adding a new dependency for every small utility function that is used. I'd prefer to just implement `capitalize()` here.